pkgs/applications/window-managers/volare/default.nix

```diff
@@ -0,0 +1,95 @@
+{ lib, stdenv, fetchFromGitea, fetchpatch, substituteAll, swaybg, sway
+, meson, ninja, pkg-config, wayland-scanner, scdoc, python3
+, wayland, libxkbcommon, pcre2, json_c, libevdev
+, pango, cairo, libinput, gdk-pixbuf, librsvg
+, wlroots_0_16, wayland-protocols, libdrm
+, nixosTests
+# Used by the NixOS module:
+, isNixOS ? false
+, enableXWayland ? true, xorg
+, systemdSupport ? lib.meta.availableOn stdenv.hostPlatform systemd, systemd
+, trayEnabled ? systemdSupport
```

Comment:
Please format these one per line; otherwise rebasing treewides in the future will be a massive headache.

You can put two parameters on the same line if one of them is a flag which influences the other (like `systemdSupport` and `systemd`).